code/modules/mob/living/simple_animal/abnormality/he/white_lake.dm

```diff
@@ -81,6 +151,24 @@
 	H.put_in_hands(wep) 		//Time for pale
 	sword = TRUE
 
+/mob/living/simple_animal/hostile/abnormality/whitelake/breach_effect(mob/living/carbon/human/user)
```

Comment:
Quick heads up, you put this inside the code for the champion

Reply:
I do not understand what you mean by this, did you mean in the "section" that champion stuff is handled? This applies to her "breach" which is why it's next to "teleport". I can move it if you wish.